paddle/fluid/framework/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ add_subdirectory(details)
 add_subdirectory(fleet)
 add_subdirectory(io)
 add_subdirectory(new_executor)
+add_subdirectory(cinn)
```

Comment:
I think the name of this subdirectory is not clear enough to indicate we are the user of cinn and link it, someone like` pd2cinn` or `compiler_adaptor` maybe better.

Reply:
Agree, but rename to `paddle2cinn` to match naming style of other modules.